src/containers/Tenant/Schema/SchemaViewer/columns.tsx

```diff
@@ -112,12 +129,11 @@ export function getExternalTableColumns(): SchemaColumn[] {
 export function getColumnTableColumns(): SchemaColumn[] {
     return [idColumn, keyColumn, nameColumn, typeColumn, notNullColumn];
 }
-export function getRowTableColumns(extended: boolean): SchemaColumn[] {
+export function getRowTableColumns(extended: boolean, hasAutoIncrement: boolean): SchemaColumn[] {
     const rowTableColumns = [idColumn, keyColumn, nameColumn, typeColumn, notNullColumn];
 
-    if (extended) {
-        return rowTableColumns.concat(familyColumn, mediaColumn, compressionColumn);
-    }
+    if (extended) rowTableColumns.push(familyColumn, mediaColumn, compressionColumn);
```

Comment:
Lets not use such a shorthands in one row, cause all over the code we use it with curly braces.
When we move to new linter major, we'll add a rule to avoid it.

Reply:
I've done. Do you mean  eslint curly: error?